index.html

```diff
@@ -478,6 +591,130 @@ <h5 class="card-header-text">Web Development</h5>
     </div>
         </section>
 
+        <section class="video">
+            <div class="container">
+                <div class="section-header" id="clients-header">
+                    <h2 class="section-header-title">We are providing best business service.</h2>
+                    <p class="section-header-text">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics </p>
+                </div>
+                <div class="content-video">
+                    <video class="video" controls poster="img/cover-video.jpg">
+                        <source src="https://media.geeksforgeeks.org/wp-content/uploads/20200409094356/Placement100-_-GeeksforGeeks2.mp4" type="video/mp4">
+                    This video is not supported by your browser.
+                    </video>
+                    <div class="playpause"></div>
+                </div>
+            </div>
+        </section>
+
+        <section class="clients">
+            <div class="container">
+                <div class="section-header" id="clients-header">
+                    <h2 class="section-header-title">What Clients Say</h2>
+                    <p class="section-header-text">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics </p>
+                </div>
+                        <div class="grid-cards">
```

Comment:
I think it would be good to add the border-radius to grid-cards: 5px; (same as in the design). 
Regarding the padding 80px everywhere - I think it`s too much and maybe will look better padding: 50px 0 80px 0;

As for me, everything else is fine!

Reply:
oh true, thank you :)

Reply:
Adding the same padding is a clever move.